test/options/form.test.js

```diff
@@ -0,0 +1,75 @@
+const assert = require('chai').assert;
+const { Builder, By, until } = require('selenium-webdriver');
+const test = require('selenium-webdriver/testing');
+
+test.describe('Options select form', function() {
+  this.timeout(50000);
+
+  const addr = 'http://localhost:3000';
+  let driver;
+
+  test.beforeEach(() => {
+    driver = new Builder()
+      .forBrowser('chrome')
+      .build();
+  });
+
+  test.afterEach(() => {
+    driver.quit();
+  });
+
+  test.xit('Loads', () => {
+    login(driver);
+    driver.findElement(By.id('logout'))
+      .getText()
+      .then(text => {
+        assert.equal(text, 'Log Out');
+      });
+  });
+
```

Comment:
I think they look fine. You could refactor some of your arrow functions into one line and drop the `{}`. You could also use object syntax on some of the "findby's". Instead of `By.id('logout')` you can simply write `{id: 'logout'}. What do you feel is clunky or imprecise about them?